src/c++/vdo/base/vdo.h

```diff
@@ -274,10 +278,12 @@ struct vdo {
 	u64 starting_sector_offset;
 	struct volume_geometry geometry;
 
+#if defined(VDO_INTERNAL) || defined(INTERNAL)
 	/* For sysfs */
 	struct kobject vdo_directory;
 	struct kobject stats_directory;
 
+#endif
 	/* N blobs of context data for LZ4 code, one per CPU thread. */
 	char **compression_context;
```

Comment:
As long as you're fixing whitespace, can you shift the #ifdef for data_vio-count (line 291) to get rid of that double blank line, too?